tests/acceptance/api_test.go

```diff
@@ -978,11 +978,6 @@ func TestApi(t *testing.T) {
 
 						_, err := dbSvc.TransitionAccountStatus(accountID, db.Leased, db.Ready)
```

Comment:
@kddejong I'm guessing you'll get a linting error here, on the unhandled error.

Can just change this to:

```
_, _ = dbSvc.TransitionAccountStatus(accountID, db.Leased, db.Ready)
```

Reply:
Good catch.... bad cut and paste.